docs/4-user-guide/2-zarf-packages/3-zarf-init-package.md

```diff
@@ -0,0 +1,142 @@
+| zarf-seed-registry      | Adds a temporary container registry so Zarf can bootstrap itself into the cluster.                                              |
+| zarf-registry           | Adds a long-lived container registry service&mdash;[docker registry](https://docs.docker.com/registry/)&mdash;into the cluster. |
+
+:::tip
+
+You can further customize how the registry behaves by setting variables such as `REGISTRY_PVC_SIZE` with a config file or `--set` on `zarf init`.
+
+To see a full list of `variables` you can view the [zarf.yaml that defines the registry](https://github.com/defenseunicorns/zarf/blob/main/packages/zarf-registry/zarf.yaml).
+
+:::
+
+Beyond the registry, their are also fully-optional components available for the init package.  Many of these also have external configurations you can set with `zarf init` (such as `--git-*`), but these components provide an easy way to get started in environments where these core services are needed and may not already exist.
+
+| Components   | Description                                                                                                                                                       |
+| ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| k3s          | REQUIRES ROOT (not sudo). Installs a lightweight Kubernetes Cluster on the local host&mdash;[K3s](https://k3s.io/)&mdash;and configures it to start up on boot.   |
+| logging      | Adds a log monitoring stack&mdash;[promtail/loki/grafana (aka PLG)](https://github.com/grafana/loki)&mdash;into the cluster.                                      |
+| git-server   | Adds a [GitOps](https://about.gitlab.com/topics/gitops/)-compatible source control service&mdash;[Gitea](https://gitea.io/en-us/)&mdash;into the cluster. |
+
+There are two ways to deploy these optional components. First, you can provide a comma-separated list of components to the `--components` flag, such as `zarf init --components k3s,git-server --confirm`, or, you can choose to exclude the `--components` and `--confirm` flags and respond with a yes (`y`) or no (`n`) for each optional component when interactively prompted.
+
+:::caution
+
+Deploying the 'k3s' component will require `root` access (not just `sudo`), as it modifies your host machine to install the cluster and by default only provides access to the cluster to the `root` user.
+
+:::
+
+:::tip
+
+The `k3s` component included in Zarf differs from the default `k3s` install in that it disables the installation of `traefik` out of the box.  This was done so that people could more intentionally choose if they wanted `traefik` or another ingress provider (or no ingress at all) depending on their needs.  If you would like to return `k3s` to its defaults, you can set the `K3S_ARGS` zarf variable to an empty string:
+
+```
+# zarf init --components k3s --set K3S_ARGS="" --confirm
```

Comment:
```suggestion
# example init + providing no arguments to K3S setup
zarf init --components k3s --set K3S_ARGS="" --confirm
```

Reply:
The # here is actually the shell prompt https://stackoverflow.com/questions/41930997/what-is-the-name-of-the-or-signs-that-indicates-if-youre-root - might be a little esoteric and we could make it clearer with more prompt-like indicators but # is for a root prompt and $ is for a non-root one.

Reply:
IK that, but most users will not know that.

Perhaps `root@machine:~#` would help drive that home.